stylesheets/_layout.tabs.scss

```diff
@@ -88,6 +89,7 @@
 }
 
 .tabs__content {
+    height: 100%;
```

Comment:
@jadu/pulsar ⚠️ Possibility of unintended consequences, had no effect in lexicon bit shout up if this concerns you

Reply:
Checked it in CMS and down to IE9. It seems to work properly without breaking anything.

Reply:
Yeah, this looks potentially concerning. Will need to test in CXM.

Reply:
Tested in CXM all good.